2022/_data/jobs.yml

```diff
@@ -147,5 +147,69 @@
+        "link": "https://careers.ingrammicro.com/es/search/jobs?q=&location=barcelona"
+      }
+    ]
+  },
+  {
+    "company": {
+      "name": "Allianz Technology SE",
+      "logo-url": "/assets/img/sponsors/allianz-technology.png",
+      "description": "Allianz Technology is the global IT service provider for Allianz and delivers IT solutions that drive the digitalization of the Group. With more than 10,000 employees located in 55 countries around the globe",
+      "level": "regular"
+    },
+    "links": {
+      "web": "https://tech.allianz.com/en.html"
+    },
+    "vacancies": [
+      {
+        "title": "DevOps Expert",
+        "location": "Barcelona",
+        "description": "<strong>KEY RESPONSIBILITIES</strong><br/>- Defines how to automate processes as well as deployment tasks. Administrates our cloud infrastructures based on AWS, Azure and private cloud.<br/>- Sets up CI/CD pipelines and educates the service teams in order to enforce a DevOps transformation.<br/>- Works with architect(s) and various parties to design & build the operational processes in DevOps manner based on enterprise guidelines and standards.<br/>- Actualizes the DevOps processes for environment management including infrastructure, platform, and application lifecycle.<br/>- Participates actively in knowledge sharing activities like communities of practice, and/or other knowledge exchanges.<br/>",
```

Comment:
It's the same offer twice?

Reply:
No, there was  a duplicated job offer because of the mess in the excel. It is fixed now: DevOps here and IT Solutions Architect below